integration/types/datatype_test.rb

```diff
@@ -208,15 +212,14 @@ def test_can_insert_all_collection_types_nil_values
       parameters = ["zz"]
       DatatypeUtils.collection_types.zip('a'..'z').each do |collection_type, letter1|
         DatatypeUtils.primitive_datatypes.zip('a'..'z').each do |datatype, letter2|
-          unless collection_type == 'Tuple'
-            parameters.push("#{letter1}_#{letter2}")
-          end
+          parameters.push("#{letter1}_#{letter2}")
         end
       end
 
       arguments = []
-      46.times { arguments.push('?') }
+      (DatatypeUtils.collection_types.size*DatatypeUtils.primitive_datatypes.size+1).times { arguments.push('?') }
 
+      #p "INSERT INTO simplex.mytable (#{parameters.join(",")}) VALUES (#{arguments.join(",")})"
```

Comment:
Was this intentionally left in? If so, please add a little more commenting around why.